src/components/FilterBar.test.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+import queryString from 'query-string';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import FilterBar from './FilterBar';
+
+const renderWithTheme = (ui) =>
+  render(<ThemeProvider theme={createTheme()}>{ui}</ThemeProvider>);
+
+const queryOptions = {
+  arrayFormat: 'bracket-separator',
+  arrayFormatSeparator: ',',
+};
+
+const params = queryString.parse(
+  { sortBy: 'amount.asc' },
+  {
+    parseNumbers: true,
+    ...queryOptions,
+  }
+);
+
+const setQueryParam = () => {};
+
+describe('HomeSection', () => {
+  test('renders the component', () => {
+    renderWithTheme(<FilterBar queryParams={params} {...{ setQueryParam }} />);
+  });
+  test('checks the number of buttons', () => {
```

Comment:
```suggestion
test('Amount filter rendered', () => {
```

Reply:
^ I think that's supposed to be the name of the test below.

This could be a `"renders amount filter and sorting dropdown by default"` and you could add `expect` calls that additionally make sure each are rendered and the default sorting value is set.